packages/plugin-starknet/src/providers/portfolioProvider.ts

```diff
@@ -0,0 +1,178 @@
+import {
+    elizaLogger,
+    IAgentRuntime,
+    Memory,
+    Provider,
+    State,
+} from "@ai16z/eliza";
+
+import { fetchWithRetry, getStarknetAccount } from "../utils";
+import { ERC20Token } from "../utils/ERC20Token";
+
+const CONFIG = {
```

Comment:
lets move this to a tokens.ts file

Reply:
Moved `PORTFOLIO_TOKENS` to the existing token.ts